src/components/Callout/Callout.tsx

```diff
@@ -22,16 +25,51 @@ export const Callout: React.FC<CalloutProps> = ({
   onClick = () => void 0,
   ...props
 }) => {
-  useTimeout(onClick, timeout);
+  const [isOpened, setIsOpened] = useState(false);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setIsOpened(true);
+    }, 150);
+
+    return () => {
+      clearTimeout(timeoutId);
+    };
+  }, []);
+
+  const handleRetryClick = () => {
+    setIsOpened(false);
+    const timeoutId = setTimeout(() => {
```

Comment:
it closes then waits a while before calling onClick?

Reply:
Eeyup, in that way we can first of all fire an animation, and then unmount the component.